docs/changes.rst

```diff
@@ -9,9 +9,9 @@
 0.11.0
 ------
 
-- In book search to provide query_type parameterization for AND vs OR queries (#87)
+- In book search: query_type parameterization for AND vs OR queries (#87)
 - Fix number of migrations to rollback in ci_test_migrations.sh
-- get only the highest version for each book a page is in, return full ident_hash, as well as authors. Put same-as-page-authors first, since this is likely to be the orginal book the page was published in.  Returned as list of hashes in page content-extras
+- Return info about what books a page is included in as well in content-extras
```

Comment:
Maybe this should be a separate changeset for another pull request? It seems a little out of scope, no?